website/docusaurus.config.js

```diff
@@ -246,6 +242,9 @@ var siteSettings = {
           //showLastUpdateAuthor: false,
 
           sidebarCollapsible: true,
+          exclude: [
+            'hover-terms.md'
```

Comment:
Prevents `hover-terms` page from being built on the site. This file is only used for storing hover snippets which are used in the custom `Term` component.